01-managing-files-with-ganga.md

```diff
@@ -246,8 +246,8 @@ j2.outputfiles.get(DiracFile)[0].get(localPath='.')
 ```
 
 > ## Using Ganga v600r44 {.callout}
-> Using `DiracFile` with the `Local` backend doesn't work with Ganga v601r19.
-> You can try again with Ganga v600r44, that happens to work in this instance, 
+> Using `DiracFile` with the `Local` backend has had issues in recent versions.
+> If this does not work, you can try again with Ganga v600r44, that happens to work in this instance, 
```

Comment:
The reference to 6.0.44 should be removed as the release will disappear very soon. If there is some problem present with newer releases it should be fixed.